tests/landing_page/landing_page.js

```diff
@@ -0,0 +1,116 @@
+var file_path = process.env.FILE_PATH;
+var webdriver = require('selenium-webdriver');
+var assert = require('chai').assert;
+var { Builder, By } = require("selenium-webdriver")
+const phantomcss = require('phantomcss');
+
+describe("Testing Scholars Project", function () {
+    this.timeout(3 * 1000 * 60);
+    var driver = new webdriver.Builder()
+        .forBrowser('chrome')
+        .build();
+
+        before(function setupWebdriver(done) {
+            var file_url = `file://${file_path}`;
+            driver.get(file_url).then(done);
+        });
+
+        after(function() {
+            driver.quit();
+        });
+
+        it('test case: FoodBag Logo is present', async () => {
+            const name = await driver.findElement(By.css("body"));
+            const text = await name.getText();
+            assert.include(text, "Food", `Ensure "food bag" text is present on header`);
+        });
+
+        it('test case: FoodBag Logo is present', async () => {
+            const name = await driver.findElement(By.css("body"));
+            const text = await name.getText();
+            assert.include(text, "Bag", `Ensure "food bag" text is present on header`);
+        });
+
+        it('test case: FoodBag text is present on the page', async () => {
+            const name = await driver.findElement(By.css("body"));
+            const text = await name.getText();
+            assert.include(text, "FoodBag", `Ensure "FoodBag" text is present`);
+        });
+
+        it("test case: check service-title", async () => {
+            const element = await driver.findElement(By.css("body"))
+            const text = await element.getText();
+            assert.include(text, "Anything food, we have you covered.", "Ensure Quantity label is present");
+        })
+
+        it("test case: check service-title", async () => {
+            const element = await driver.findElement(By.css("body"))
+            const text = await element.getText();
+            assert.include(text, "Where are you?", "Ensure Quantity label is present");
+        })
+
+        it("test case: quantity field is a text input field", async () => {
+            const element = await driver.findElement(By.css("body"))
+            const text = await element.getText();
+            assert.include(text, "Pick an interest", "Ensure Quantity label is present");
+        })
+
+        it("test case: check service-title", async () => {
```

Comment:
Do you want to maybe number this service titles?  so it is different for each one and the  scholar can know which is missing.